honeybadgermpc/preprocessing.py

```diff
@@ -4,6 +4,7 @@
 from uuid import uuid4
 from random import randint
 from os import makedirs
+from gmpy2 import num_digits
```

Comment:
This is an overkill dependency just to use `num_digits` given that we are otherwise only using an int

Reply:
Good call-- TIL `int` has a `bit_length()` method already.